environment-pipelines/locals.tf

```diff
@@ -7,9 +7,12 @@ locals {
 
   stage_config = yamldecode(file("${path.module}/stage_config.yml"))
 
+  env_config    = { for name, config in var.environment_config : name => merge(lookup(var.environment_config, "*", {}), config) }
+  enriched_envs = [for name, env in var.environments : merge(lookup(local.env_config, name, {}), env, { "name" = name })]
```

Comment:
What is enriched about it? Is it worth having a comment to explain?

Reply:
Is it worth adding an output for enriched_envs also, so we could check the structure in the terminal when doing a terraform plan?